.eslintrc

```diff
@@ -49,32 +58,60 @@
         "no-unused-expressions": 2,
         "no-with": 2,
         "radix": 2,
+        "vars-on-top": 1,
```

Comment:
I know that in some cases I'm defining variables right were I need them (in JS). In my opinion this can be usefull in longer functions, where the var is just needed for a few lines.

As it is a convention to define all variables at the top of codeblock I'm totally ok with this anyhow.

@bentrm Do you know if its ok to have empty lines between the variable definitions?